astropy/wcs/tests/test_utils.py

```diff
@@ -57,6 +58,40 @@
 from astropy.wcs.wcsapi.fitswcs import SlicedFITSWCS
 
 
+def test_north_pole_angle():
+    # create basic wcs with north up
+    mywcs = WCS(naxis=2)
+    mywcs.wcs.crval = [1, 1]
+    mywcs.wcs.cdelt = [0.1, 0.1]
+    mywcs.wcs.crpix = [1, 1]
+    mywcs._naxis = [1000, 500]
+    mywcs.wcs.ctype = ["RA---TAN-SIP", "DEC--TAN-SIP"]
+
+    # simplest test - north is 90 degrees from positive x
+    pix = (0, 0)
+    assert np.isclose(north_pole_angle(pix, mywcs), 90 * u.deg, rtol=1e-5)
```

Comment:
Use this to compare Quantity. This comment applies to all other similar occurrences.

https://github.com/astropy/astropy/blob/e27b22432d1582f20dc15135afb116e5b3a1a165/astropy/tests/helper.py#L187-L187

Reply:
Hi, I am working with this team and have some questions about this suggestion. Can we make it a requirement that the image be flattened into 2D by the user before running this function? Since the purpose of the function is to compute the needed parameters for the matplotlib AnchoredDirectionArrows function that actually draws the arrows, could it be up to the user to input an image with the correct dimensionality?